week-1-ts-ecosystem/w01e01-typescript-basics/src/types/index.d.ts

```diff
@@ -0,0 +1,11 @@
+interface Address {
+  street: string;
+  no: number;
+}
+
+export interface User {
+  name: string;
+  age: number;
+  email: string;
+  address: Address;
+}
```

Comment:
To bym rozbił na osobne pliki albo jeśli w jednym to dał wyraźnie że to `model` dla naszego `User`

CIekawa koncepcja z `d.ts`  żeby tutaj się nie pomylić i nie wjechać z danymi.
Tak to jest spoko tylko nazwa `types/` zbyt ogólna.

Reply:
No właśnie się zastanawiałem czy ten katalog nie będzie zbyt ogólny, ale z drugiej strony tak prosty przykład wydawał mi się zbyt prosty, żeby aż tak dokładnie go dzielić. Przy większej aplikacji z pewnością bym to rozdzielił 🤔

Reply:
Poprawiłem 👍